docs/usage.md

```diff
@@ -64,6 +64,18 @@ For single-sample peaking calling without a control sample, leave the second col
 ### `--broad`
 Run MACS with the `--broad` flag. With this flag on, MACS will try to composite broad regions in BED12 ( a gene-model-like format ) by putting nearby highly enriched regions into a broad region with loose cutoff. The broad region is controlled by the default qvalue cutoff 0.1.
 
+### `--saturation`
+Running saturation analysis by sub-sampling the sequence reads of ChIP sample from 10% to 100% with 10% interval, then calling peaks with MACS.
+
+### `--blacklist_filtering`
+Filtering ENCODE blacklisted regions from ChIP-seq peaks.
+
+### `--blacklist`
+If you prefer, you can specify the full path to the blacklist regions (should be in .BED format) to be filtered out when you run the pipeline:
```

Comment:
Specify ChIP-seq peaks so it's clear that we're not filtering aligned reads here.

Reply:
I think you also need to specify `--blacklist_filtering` in addition for this to work? If so, specify this here and add to the example.